subprojects/packagefiles/sqlite3/meson.build

```diff
@@ -15,11 +15,43 @@ endif
 libm = cc.find_library('m', required: false)
 
 sources = files('sqlite3.c')
+
+sqlite_args = [
+  '-DSQLITE_ALLOW_ROWID_IN_VIEW',
+  '-DSQLITE_DISABLE_DIRSYNC',
+  '-DSQLITE_ENABLE_COLUMN_METADATA',
+  '-DSQLITE_ENABLE_DBSTAT_VTAB',
+  '-DSQLITE_ENABLE_DBPAGE_VTAB',
+  '-DSQLITE_ENABLE_FTS3',
+  '-DSQLITE_ENABLE_FTS3_PARENTHESIS',
+  '-DSQLITE_ENABLE_FTS3_TOKENIZER',
+  '-DSQLITE_ENABLE_FTS4',
+  '-DSQLITE_ENABLE_FTS5',
+  '-DSQLITE_ENABLE_GEOPOLY',
+  '-DSQLITE_ENABLE_LOAD_EXTENSION',
+  '-DSQLITE_ENABLE_PREUPDATE_HOOK',
+  '-DSQLITE_ENABLE_RTREE',
+  '-DSQLITE_ENABLE_SESSION',
+  '-DSQLITE_ENABLE_STMTVTAB',
+  '-DSQLITE_ENABLE_UNLOCK_NOTIFY',
+  '-DSQLITE_ENABLE_UPDATE_DELETE_LIMIT',
```

Comment:
[Documented](https://www.sqlite.org/compile.html) to be disallowed when building the amalgamation.